editor/plugins/texture_editor_plugin.cpp

```diff
@@ -70,7 +70,48 @@ void TexturePreview::_update_metadata_label_text() {
 		format = texture->get_class();
 	}
 
-	metadata_label->set_text(vformat(String::utf8("%s×%s %s"), itos(texture->get_width()), itos(texture->get_height()), format));
+	const Ref<Image> image = texture->get_image();
+	if (image.is_valid()) {
+		const int mipmaps = image->get_mipmap_count();
+		int memory = image->get_width() * image->get_height() * Image::get_format_pixel_size(image->get_format());
```

Comment:
If Godot supports 16k textures (16384 x 16384), this may signed-integer-overflow in the event that the raw texture is over 2GB of VRAM and the texture is RGBAH *8 bytes per pixel) or RGBAF (16 bytes per pixel) format:
```
>>> 16384*16384*8
2147483648
>>> 16384*16384*16
4294967296
```

That said, I wouldn't expect such large textures to work particularly well on any hardware due to fragmentation, but thought it would be worth a comment.

Reply:
Might as well cast to uint64_t before multiplying and then do the following calculations in uint64_t. Thats what ``humanize_size`` expects anyway.